### PR TITLE
Add sanity test for PyTorch SuperRes example

### PR DESCRIPTION
**Pull request by Kh4L**
Signed-off-by: Serge Panev <spanev@nvidia.com>

### CODE REVIEW

**Comment by Kh4L** [comment]
Build 681390

**Comment by Kh4L** [comment]
Build 700664

**Comment by Kh4L** [comment]
!build

**Comment by Kh4L** [comment]
!build

**Comment by Kh4L** [comment]
!build

**Comment by Kh4L** [comment]
!build

**Comment by Kh4L** [comment]
!build

**Comment by Kh4L** [comment]
!build

**Comment by Kh4L** [comment]
!build

File: qa/L1_superres_pytorch/test.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+DATA_DIR=data_dir/720p/scenes
+# Creating simple working env for PyTorch SuperRes example
+mkdir -p $DATA_DIR/train/
+mkdir -p $DATA_DIR/val/
+
+
+cp ../video_files/* $DATA_DIR/train/
+cp ../video_files/* $DATA_DIR/val/
+
+# Pre-trained FlowNet2.0 weights
```
Comment (JanuszL) [inline qa/L1_superres_pytorch/test.sh]: Could you add the reference how to get this file manually?

Comment (Kh4L) [reply qa/L1_superres_pytorch/test.sh]: The pre-trained model on the CI is actually taken from the publicly available on Google Drive https://drive.google.com/file/d/1QW03eyYG_vD-dT-Mx4wopYvtPu_msTKn/view and the link is in the [README here](https://github.com/NVIDIA/DALI/blob/0de4db3a8ad72ea6fbf6559f2c6a3bd97b789307/docs/examples/video/superres_pytorch/README.rst#flownet2-sd-implementation-and-pre-trained-model)

Comment (JanuszL) [reply qa/L1_superres_pytorch/test.sh]: I know, but if someone starts redoing tests may be not aware of this.

Comment (Kh4L) [reply qa/L1_superres_pytorch/test.sh]: Sure, just added the comment

**Comment by Kh4L** [comment]
!build

File: qa/L1_superres_pytorch/test.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+source qa/setup_dali_extra.sh
+
+cd docs/examples/video
+
+mkdir -p video_files
+
+container_path=${DALI_EXTRA_PATH}/db/optical_flow/sintel_trailer/sintel_trailer.mp4
+
+IFS='/' read -a container_name <<< "$container_path"
+IFS='.' read -a splitted <<< "${container_name[-1]}"
```
Comment (jantonguirao) [inline qa/L1_superres_pytorch/test.sh]: split

Comment (Kh4L) [reply qa/L1_superres_pytorch/test.sh]: Done

**Comment by Kh4L** [comment]
!build

**Pull request merged**